isb/Dockerfile

```diff
@@ -56,10 +34,13 @@ RUN mkdir /var/log/isamples
 RUN chmod 755 /var/log/isamples
 
 # Put the React build output inside isb_web ui directory for serving as part of the uvicorn installation
-# This statement is a bit confusing -- the --from=node_build is the build stage we are copying *from*,
-# and the /app/build is the path in *that* image (the multistage builds produce separate images).  The
-# dest path (/app/isb_web/ui) is the path in *this* image.
-COPY --from=node_build /app/build/ /app/isb_web/ui/
+# This is built as part of the web_ui github action -- then the output is checked out in the gh-pages branch, copied to 
+# the container, and we run python inside the container to search and replace the necessary values in the container itself.
+ARG ISB_SITEMAP_PREFIX=https://mars.cyverse.org
+RUN echo "ISB_SITEMAP_PREFIX IS $ISB_SITEMAP_PREFIX"
+COPY ./isamples_webui/build/ /app/isb_web/ui/
+COPY ./massage_webui_output.py .
+RUN python /app/massage_webui_output.py -u https://hyde.cyverse.org/isamples_central -r $ISB_SITEMAP_PREFIX -p /app/isb_web/ui/static/js
```

Comment:
Then run the python script in the container to fix up the source.  This appears to be running locally for me, but isn't producing the output I expect.